docs/docs/cmd/spo/group/group-member-add.md

```diff
@@ -1,11 +1,11 @@
-# spo group user add
+# spo group member add
 
 Add a user or multiple users to SharePoint Group
```

Comment:
what do you think about changing also this in the description ? 🤔

```suggestion
Add a member or multiple members to SharePoint Group
```

Reply:
Hi @Adam-it, I had left it on purpose, as the only type of members we can currently add are users. I wanted to change that when we add the azure ad group option. But okay, I'll change it anyway. 👍